assets/scss/modules/_phase-banner.scss

```diff
@@ -0,0 +1,8 @@
+.header-banner-wrapper {
+    max-width: 960px;
+    margin: 0 auto;    
+    display: block;
+    @media screen and (max-width: 986px) {
```

Comment:
In future not all devices will fit in the mobile,tablet,desktop categories. I used the exact width where the banner needs to be adjusted. This way any new device (say 977px) will adjust accordingly.